src/quickgui/qgsquickcoordinatetransformer.h

```diff
@@ -83,6 +83,9 @@ class QUICK_EXPORT QgsQuickCoordinateTransformer : public QObject
     //!\copydoc QgsQuickCoordinateTransformer::transformContext
     void setTransformContext( const QgsCoordinateTransformContext &context );
 
+    //!\copydoc QgsQuickCoordinateTransformer::transformContext
```

Comment:
Just as a general note, `\copydoc` (unfortunately) doesn't play well with sipify. But since this module is not sipified (I think) no worries :)